password/password.cabal

```diff
@@ -26,35 +32,42 @@ library
       src
   exposed-modules:
       Data.Password
+      Data.Password.Argon2
+      Data.Password.Bcrypt
+      Data.Password.PBKDF2
+      Data.Password.Scrypt
   other-modules:
       Paths_password
+      Data.Password.Internal
```

Comment:
I'm a fan of exposing these `Internal` modules for power users.

Although, we could always just wait until someone explicitly requests it.

Reply:
Everything from `Data.Password.Internal` is re-exported by `Data.Password`, except the data constructor of `Password`, but you can still get to the `Text` inside using the `unsafeShowPassword` functions.

The other utility functions are irrelevant for this package (`showT readT toBytes fromBytes`)

Reply:
Ok, let's do that.